packages/web-component/src/scss/_shame.scss

```diff
@@ -1,3 +1,11 @@
+[contenteditable][data-placeholder]:empty:before {
```

Comment:
That's funny, putting shameful code in `_shame.scss`! 
Is this based on the idea here https://csswizardry.com/2013/04/shame-css/ ?

Reply:
and since you brought my attention here, I have to ask, are any of the hacks in this file fixable by now?

Reply:
yea pretty similar to that. I'm not sure. probably, but that kind of refactoring isn't on this sprint for me :)